account_mail_templates/__manifest__.py

```diff
@@ -1,24 +1,22 @@
 # Author: Andrius Laukavičius. Copyright: JSC Focusate.
 # See LICENSE file for full copyright and licensing details.
 {
-    'name': "Invoices Custom QWeb Email Templates",
+    'name': "Invoicing Custom QWeb Email Templates",
     'version': '15.0.1.1.0',
-    'summary': 'Invoices jinja2 templates rewritten to QWeb templates',
-    'license': 'AGPL-3',
+    'summary': 'Invoicing templates using QWeb inheritance',
+    'license': 'LGPL-3',
     'author': "Focusate",
     'website': "http://www.focusate.eu",
-    'category': 'Accounting',
+    'category': 'Accounting/Reporting',
```

Comment:
```suggestion
    'category': 'Accounting/Discuss',
```

Reply:
This could still be more of a report. And `Discuss` category does not exist as secondary. So better use existing one.